caltime.h

```diff
@@ -0,0 +1,20 @@
+#ifndef CALTIME_H
+#define CALTIME_H
+
+#include "caldate.h"
+
+struct caltime {
+  struct caldate date;
+  int hour;
+  int minute;
+  int second;
+  long offset;
```

Comment:
This is used exactly once, in caltime_utc.c where it is initialized with 0, which makes me think we can just remove it.